fieldservice_recurring_repair/models/fsm_recurring.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class FSMRecurringOrder(models.Model):
+    _inherit = "fsm.recurring"
+
+    def _split_create(self, vals):
```

Comment:
what do you think of something like
```suggestion
    def _create_by_equipment(self, vals):
```
?

Reply:
I did `_split_create_by_equipment` to be totally explicit